test/integration/main_test.go

```diff
@@ -195,12 +201,12 @@ contacts:
 			}
 
 			// Check if the file uploaded exists in BigQuery.
-			queryString := fmt.Sprintf("SELECT data FROM `%s.%s.%s`", cfg.ProjectID, cfg.BigQueryDataSetID, tc.bigqueryTable)
+			queryString := fmt.Sprintf("SELECT data, CASE WHEN STRING(attributes.ProcessErr) is null THEN '' ELSE STRING(attributes.ProcessErr) END as ProcessErr FROM `%s.%s.%s`", cfg.ProjectID, cfg.BigQueryDataSetID, tc.bigqueryTable)
```

Comment:
Instead of select multiple fields, can you just select the whole bq row and compare it. Selecting multiple fields especially using CASE xxx makes the query quite difficult to understand

Reply:
+1, it is confusing.

Reply:
Now select the whole row.